packages/lib/modules/lbp/steps/useCreateLbpStep.tsx

```diff
@@ -77,36 +88,36 @@ export function useCreateLbpStep(): TransactionStep {
   const blockProjectTokenSwapsIn = userActions === 'buy_only' ? true : false
 
   const { symbol: launchTokenSymbol } = useTokenMetadata(launchTokenAddress, selectedChain)
-  const { symbol: collateralTokenSymbol } = useTokenMetadata(collateralTokenAddress, selectedChain)
+  const { symbol: reserveTokenSymbol } = useTokenMetadata(reserveTokenAddress, selectedChain)
 
   const hasRequiredValues =
-    launchTokenSymbol &&
-    collateralTokenSymbol &&
-    name &&
-    chainId &&
-    userAddress &&
-    launchTokenAddress &&
-    collateralTokenAddress &&
-    projectTokenStartWeight &&
-    reserveTokenStartWeight &&
-    projectTokenEndWeight &&
-    reserveTokenEndWeight &&
-    startTime &&
-    endTime
+    !!launchTokenSymbol &&
```

Comment:
If everything here is taken from sale structure and project forms, can we just check that the forms are valid or is there any reason to check field by field?

Reply:
How can you reach the creation modal without filling the `startTime` and `endTime` fields? Even then are they not requiered? (meaning that `saleStructureForm.formState.isValid` will be false?

Reply:
the error happens instantly as the form renders just after pasting the sale token address before entering any other details

Reply:
[issue-1176.patch](https://github.com/user-attachments/files/21039749/issue-1176.patch)

This will more or less solve it (don't know if missed any edge case). You can take a look and use it if you think it works.

Reply:
thanks for showing me how it is done i like that refactor a lot! :pray:

moving the `LbpCreationProvider` to only wrap the modal fixes so many weird quirks!